bc_obps/registration/api/operator.py

```diff
@@ -1,4 +1,4 @@
-from typing import Optional
+from typing import Optional, List
```

Comment:
I think useless list import form old work I did?

Reply:
No it's definitely needed, try removing it and run `make reset_db`